flocker/node/test/test_container.py

```diff
@@ -460,6 +460,34 @@ def test_command_line(self):
             pvector(command_line),
         )
 
+    def test_swappiness(self):
+        """
+        ``StartApplication.run()`` passes ``swappiness`` to
+        ``DockerClient.add`` which is used when creating a Unit.
+        """
+        expected_swappiness = 99
+        fake_docker = FakeDockerClient()
+        deployer = ApplicationNodeDeployer(u'example.com', fake_docker)
+
+        application_name = u'site-example.com'
+        application = Application(
+            name=application_name,
+            image=DockerImage(repository=u'clusterhq/postgresql',
```

Comment:
Why dont we use a smaller image here? Any reason we are using postgresql just to test the swappiness? Could mean longer pulls for new test nodes.